app/models/pension/charges/UnauthorisedPaymentsViewModel.scala

```diff
@@ -158,6 +158,19 @@ case class UnauthorisedPaymentsViewModel(surchargeQuestion: Option[Boolean] = No
   def toPensionsCYAModel: PensionsCYAModel = PensionsCYAModel.emptyModels.copy(unauthorisedPayments = this)
 }
 
+case class UnauthorisedPaymentsAuditViewModel(unauthorisedPayment: Option[String] = None,
```

Comment:
Should this be an enum rather than string according to the ticket? e.g. Option[UnauthorisedPayment]

Reply:
I don’t think an enum adds much value here, I'm keeping in as Option[String] for the time being